python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -746,6 +748,83 @@ def run_transformer_int4_loadlowbit_npu_win(repo_id,
     gc.collect()
     return result
 
+def run_transformers_openvino(repo_id,
+                                 local_model_hub,
+                                 in_out_pairs,
+                                 warm_up,
+                                 num_trials,
+                                 num_beams,
+                                 low_bit,
+                                 batch_size,
+                                 optimize_model):
```

Comment:
We also need to add `group_size` to these arguments, as well as add it into `config.yml`